TestsFoundation/TestsFoundation/Element/LoginFindSchool.swift

```diff
@@ -30,4 +30,8 @@ public enum LoginFindAccountResult {
     public static func item(host: String) -> Element {
         return app.find(id: "LoginFindAccountResult.\(host)")
     }
+
+    public static func item(label: String) -> Element {
+        return app.find(label: label)
+    }
```

Comment:
Does this need to be a separate function from `app.find`?

Reply:
Nope. Fine by me.